store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorage.java

```diff
@@ -653,7 +653,7 @@ private boolean changeGraphId(final String graphId, final String newGraphId, fin
                     federatedStoreCache.addGraphToCache(newGraphSerialisable, key, true);
                 } catch (final CacheOperationException e) {
                     String s = "Contact Admin for recovery. Error occurred updating graphId. GraphStorage=updated, Cache=outdated graphId.";
-                    LOGGER.error(s + " graphStorage graphId:{} cache graphId:{}", newGraphId, graphId);
+                    LOGGER.error(s ,"{} graphStorage graphId:{} cache graphId:{}", newGraphId, graphId);
```

Comment:
This needs to be fixed like you did on line [577](https://github.com/gchq/Gaffer/pull/2651/files#diff-f63625e74223d0e72648e1658bf66b918b576b71f5228eba98044b277155013aR577).